src/apps/pages/models/HealthCareModels/BrainTumorModel/brainTumorModel.py

```diff
@@ -29,24 +29,27 @@ def do_test(img):
   return res
 
 def brainTumorModel():
-  st.write("Please Upload MRI Scan of Brain")
-  codn,img = get_mri()
-  res = None
-
-  if codn!=False:
-    res = do_test(img)
-
-  if res!=None:     
-    if res == 0:
-      st.error(f'Hi {st.session_state.name},\nYou are diagnosed with Glioma.\nPlease consult a doctor.')
-      Speak(f'Hi {st.session_state.name},You are diagnosed with Glioma. Please consult a doctor.')
-    elif res == 1:
-      st.error(f'Hi {st.session_state.name},\nYou are diagnosed with Meningioma.\nPlease consult a doctor.')
-      Speak(f'Hi {st.session_state.name},You are diagnosed with Meningioma. Please consult a doctor.')
-    elif res == 2:
-      st.success(f'Congrats {st.session_state.name}, You are not diagnosed with brain tumor.')
-      Speak(f'Congrats {st.session_state.name}, You are not diagnosed with brain tumor.')
-    elif res == 3:
-      st.error(f'Hi {st.session_state.name},\nYou are diagnosed with Pituitary tumor. Please consult a doctor.')
-      Speak(f'Hi {st.session_state.name},You are diagnosed with Pituitary tumor. Please consult a doctor.')
+    if "name" not in st.session_state:
+        st.session_state.name = "User"
```

Comment:
Remove this session state `name`. Directly use 'User' or use other comment on the result.

Reply:
I Removed the st.session.state.name.
And now I am Directly using `User`.